bin/runtime/src/lib.rs

```diff
@@ -110,10 +112,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("aleph-node"),
     impl_name: create_runtime_str!("aleph-node"),
     authoring_version: 1,
-    spec_version: 56,
+    spec_version: 59,
```

Comment:
This should be current mainnet + 1. Reviewers, please confirm :P

Reply:
that is correct, at least before we make it more intelligent :P

Reply:
Agree